### ISSUE TITLE
Changelog and Requirements are Missing in Latest Update

### ISSUE THREAD

**Issue by dkarakas1** [F5Networks/bigsuds#8]
Couple of issues here:

1. Currently we're up to 1.0.5.  This is reflected in the bigsuds.py file and noted at   https://pypi.python.org/pypi/bigsuds/1.0.5.  This is not reflected in the changelog.
    
2. BigSuds v1.0.5 requires 'setuptools' >=1.4.  However, setuptools version 1.4 is _not_ available in the CentOS 7 standard repositories and requires additional repos be added/enabled to satisfy this requirement.  How to update this is discussed here: https://packaging.python.org/guides/installing-using-linux-tools/

    _When requirements change, this should be reflected in the docs somewhere..._

3. This repo is stale.  No useful Readme.MD data.  No Examples of use, no docs at 'readthedocs.org', nothing to help users leverage this project.

**Comment by caphrim007** [comment]
@dkarakas1 this is a stale repo because SOAP is not the recommended method for managing devices. It was during the 9.x - 11.x days, but the REST interface is where our focus is these days.

Since this repo is deliberately not being worked on, save the miscellaneous bugfixes to keep it moving on later versions of python, there is no testing or validation done on it; nor was there ever.

The repo is not stale. You see me answering your question here. There is no documented usage of it anywhere because that documentation exists on F5's SOAP API website here

https://devcentral.f5.com/wiki/iControl.APIReference.ashx?lc=1

In a nutshell, I leave this up here for those who are feeling lucky and to support the legacy Ansible modules.
